README.md

```diff
@@ -1,29 +1,53 @@
 # Alchemy
 
+
+[![Hex.pm](https://img.shields.io/hexpm/v/alchemy.svg?style=flat-square)](https://hex.pm/packages/alchemy)
+[![Hex.pm](https://img.shields.io/hexpm/dt/alchemy.svg?style=flat-square)](https://hex.pm/packages/alchemy)
+
 Perform refactoring experiments in production. Based on [Scientist](https://github.com/github/scientist)
 
+---
+
+## Installation
+
+1. Add alchemy to your list of dependencies in `mix.exs`:
+``` elixir
+def deps do
+  [{:alchemy, "~> 0.0.1"}]
+end
+```
+
+2. Ensure that alchemy is started before your application:
+``` elixir
+def application do
+  [applications: [:alchemy]]
+end
+```
+
 ## Perform some experiments
 
 Lets say that you have a controller that returns a list of users, and you want to change how that list of users is fetched from the database. Unit tests can help you but they can only account for examples that you've currently considered. Alchemy allows you to test your new code live in production.
 
+In these examples, the functions are imported from `Alchemy.Experiment`. If you want, you can use these functions without importing.
+
```

Comment:
No worries, just thought it'd clarify. But since the example has the `import Alchemy.Experiment`, it could be redundant.